### PR TITLE
Fix Sidebar Containers

### PR DESCRIPTION
**Pull request by theilman7**
Creating PR to begin discussion on the sidebar. I added two content containers for content in the sidebar. One is a standard container, where items can just be placed in, the other is an option for a gray background. Let me know if this is heading int he right direction.

### CODE REVIEW

File: src/styles/partials/template/_sidebar.scss [hunk 1]
```diff
@@ -18,7 +18,15 @@
   p {
     font-family: $default-heading-font;
     font-size: $smallest-heading-font-size;
-    margin: 0 0 $default-margin 0;
+  }
+
+  .dg_sidebar__content {
```
Comment (martypowell) [inline src/styles/partials/template/_sidebar.scss:23]: @theilman7 have you adjusted this markup anywhere?

Comment (theilman7) [reply src/styles/partials/template/_sidebar.scss:23]: @martypowell Not yet, just with inspect. I can copy this over and insert it on to the components page to test. Will follow up when that has been added shortly.

Comment (theilman7) [reply src/styles/partials/template/_sidebar.scss:23]: @martypowell Pushed up some corrections and staged on the components page in beta.

File: src/styles/partials/components/_summary-list.scss [hunk 2]
```diff
@@ -32,21 +28,16 @@
...
   h3 {
     color: $black;
   }
 
   i {
-    display: block;
     text-align: center;
     font-size: $h1-font-size;
-    width: 25%;
+    margin-right: $default-padding;
```
Comment (martypowell) [inline src/styles/partials/components/_summary-list.scss:40]: this is  agood example why we should have one value for spacing margn / padding. which i think should have been 15px. That would be a big refactor now, so we can leave this as is.

File: src/styles/partials/template/_sidebar.scss [hunk 3]
```diff
@@ -20,6 +20,15 @@
     font-size: $smallest-heading-font-size;
   }
 
+  .dg_sidebar__content {
+    padding: $padding-small $padding-large;
+  }
+
+  .dg_sidebar__content_gray {
```
Comment (martypowell) [inline src/styles/partials/template/_sidebar.scss]: how does this differ from the rule above this? I think maybe we could name this a little better to describe it.

Comment (theilman7) [reply src/styles/partials/template/_sidebar.scss]: Essentially all it does is add the option for a gray background. Maybe .dg_sidebar___content_gray_background? A bit long though.

**Review by martypowell — APPROVED** [review]

**Pull request merged**